docs/index.md

```diff
@@ -36,12 +36,12 @@ For instructions on how to build and run SOLA and its components, see [Getting S
 The SOLA repository contains multiple subprojects:
 
 - **SOLA (Standalone)** is the framework that initializes and uses different components to offer a Service Discovery and Publish-Subscribe API.
-- **[MINHTON](management_overlay/introduction.md)** is a peer-to-peer management overlay which organizes peers in an optimized tree structure. MINHTON is a SOLA component for service discovery. 
-- **[natter](natter_lib/introduction.md)** is a framework for peer-to-peer Publish-Subscribe algorithms.
+- **[MINHTON](management_overlay/introduction.md)** is a peer-to-peer management overlay which organizes peers in an optimized tree structure. MINHTON is a SOLA component for service discovery.
+- **[natter](natter/index.md)** is a framework for peer-to-peer Publish-Subscribe algorithms.
 The MINHCAST algorithm to broadcast a message to all nodes in a MINHTON network is included in natter.
 natter is a SOLA component for Publish-Subscribe.
 - **SolaNet** is a basic asynchronous UDP networking and utility library.
-- **[OptiMaFlow](applications/optimaflow/getting_started.md)** is an application in the domain of intralogistics that uses SOLA to schedule transport tasks to autonomous mobile robots.
+- **[OptiMaFlow](optimaflow/introduction.md)** is an application in the domain of intralogistics that uses SOLA to schedule transport tasks to autonomous mobile robots.
```

Comment:
Currently OptiMaFlow depends on SOLA and cannot be used without it. This may change in the future, but as SOLA is a hard dependency right now, I would prefer to keep it like this.
Any other thoughts?